atlasdb-client/src/main/java/com/palantir/atlasdb/schema/generated/SweepPriorityTable.java

```diff
@@ -1151,6 +1154,16 @@ public void delete(Iterable<SweepPriorityRow> rows) {
         return transformed;
     }
 
+    @Override
+    public Iterator<Map.Entry<SweepPriorityRow, SweepPriorityNamedColumnValue<?>>> getRowsColumnRange(Iterable<SweepPriorityRow> rows, ColumnRangeSelection columnRangeSelection, int batchHint) {
+        Iterator<Map.Entry<Cell, byte[]>> results = t.getRowsColumnRange(getTableRef(), Persistables.persistAll(rows), columnRangeSelection, batchHint);
+        return Iterators.transform(results, e -> {
+            SweepPriorityRow row = SweepPriorityRow.BYTES_HYDRATOR.hydrateFromBytes(e.getKey().getRowName());
+            SweepPriorityNamedColumnValue<?> colValue = shortNameToHydrator.get(PtBytes.toString(e.getKey().getColumnName())).hydrateFromBytes(e.getValue());
+            return new AbstractMap.SimpleEntry<SweepPriorityRow, SweepPriorityNamedColumnValue<?>>(row, colValue);
```

Comment:
Might consider `Maps.immutableEntry` To prevent setValue (and avoid some generics cruft).